pyproject.toml

```
@@ -0,0 +1,102 @@
+[project]
+name = "mpas_analysis"
+dynamic = ["version"]
+authors = [
+    { name="Xylar Asay-Davis", email="xylar@lanl.gov" },
+    { name="Carolyn Begeman" },
+    { name="Phillip J. Wolfram" },
+    { name="Milena Veneziani" },
+    { name="Darin Comeau" },
+    { name="Steven Brus" },
+    { name="Luke Van Roekel" },
+    { name="Anirban Sinha" },
+    { name="Riley Brady" },
+    { name="Irena Vankova" },
+    { name="Althea Denlinger" },
+    { name="Elizabeth Hunke" },
+    { name="Stephen Price" },
+    { name="Adrian K. Turner" },
+    { name="Kevin Rosa" },
+    { name="Matt Hoffman" },
+    { name="Mark Petersen" },
+    { name="Charles Doutriaux" },
+    { name="Alexander Hager" },
+]
+description = """\
+	Analysis for Model for Prediction Across Scales (MPAS) simulations.\
+	"""
+license = {file = "LICENSE"}
+readme = "README.md"
+requires-python = ">=3.9"
+classifiers = [
+    # these are only for searching/browsing projects on PyPI
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+
+
+	"Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+	"Intended Audience :: Science/Research",
+	"Topic :: Scientific/Engineering",
```

Some whitespace issues:
```suggestion

    "Development Status :: 5 - Production/Stable",
    "License :: OSI Approved :: BSD License",
    "Operating System :: OS Independent",
    "Intended Audience :: Science/Research",
    "Topic :: Scientific/Engineering",
```

---

Whoops, looks like I accidentally mixed tabs and spaced. I'll fix that.